pkg/platform/abstract/platform_test.go

```diff
@@ -1171,6 +1171,79 @@ func (suite *AbstractPlatformTestSuite) TestValidateNodeSelector() {
 	}
 }
 
+func (suite *AbstractPlatformTestSuite) TestValidatePriorityClassName() {
+	for idx, testCase := range []struct {
+		name                            string
+		priorityClassName               string
+		validFunctionPriorityClassNames []string
+		shouldFailValidation            bool
+	}{
+
+		// happy flows
+		{
+			name: "Sanity",
+			priorityClassName: "low-priority",
+			validFunctionPriorityClassNames: []string{"low-priority"},
+		},
+		{
+			name: "MultipleValidValues",
+			priorityClassName: "low-priority",
+			validFunctionPriorityClassNames: []string{"low-priority", "medium-priority", "high-priority"},
+		},
```

Comment:
add another case where `validFunctionPriorityClassNames` is `nil`
all `priorityClassName` should be valid when `validFunctionPriorityClassNames` is `nil`